src/aiogram_dialog/utils.py

```diff
@@ -174,3 +176,18 @@ def remove_indent_id(callback_data: str) -> Tuple[Optional[str], str]:
         intent_id, new_data = callback_data.split(CB_SEP, maxsplit=1)
         return intent_id, new_data
     return None, callback_data
+
+
+def sync_scroll(scroll_id: str) -> Callable[
```

Comment:
Probably, we should to put this function to widgets next to BaseScroll, not in common utils.